### PR TITLE
Documentation Spring Cleaning

### PR DESCRIPTION
**Pull request by inkdeep**
I did some polishing on documentation and added some clarifications to :additional_members behavior.

### CODE REVIEW

File: lib/declarative_authorization/in_controller.rb [hunk 1]
```diff
@@ -369,6 +369,20 @@ module Authorization
       #   end
       # The +additional_+* options add to the respective CRUD actions,
       # the other options replace the respective CRUD actions.
+      #    filter_resource_access :member => { :extra => :new }
```
Comment (stffn) [inline lib/declarative_authorization/in_controller.rb:373]: We need to be careful here.  :member, :new, :collection (and the :additional_*) take the key in the hash as the privilege to check for on the method.  That is, :member => {:extra => :new} would say :extra is a member action and please require the privilege :new to be assigned to the current user.  Maybe a more practical example is even better here: e.g. :member => {:photo => :read} for :employees, saying: there is only one member action in this controller, namely :photo. :additional_member => {:photo => :read} would instead add :photo to the other member actions :update etc.

File: lib/declarative_authorization/in_controller.rb [hunk 2]
```diff
@@ -369,6 +369,20 @@ module Authorization
...
+      # Would replace :+new+ with the method :+extra+ and :+new+ would become
+      # a member method and any calls to :+new+ would expect a params[:id] or 
+      # in the case of the above example a params[:company_id] due to the 
+      # :+nested_in+ option being set.
+      #    filter_resource_access :additional_member => { :extra => :new }
+      # Would make a method called :+extra+ that would behave just like :+new+
+      # and call the +new_branch_from_params+ before_filter.
+      #
```
Comment (stffn) [inline lib/declarative_authorization/in_controller.rb:380]: the behavior as "new" action refers to the :new and :additional_new parameter

**Comment by stffn** [comment]
Thanks for the correction.  If we get the :member stuff right, I gladly merge this in.  Would be even better as one single commit.

File: lib/declarative_authorization/in_controller.rb [hunk 3]
```diff
@@ -463,8 +477,7 @@ module Authorization
           :nested_in  => nil,
         }.merge(options)
 
-        new_actions = actions_from_option(options[:new]).merge(
-            actions_from_option(options[:additional_new]))
+        new_actions = actions_from_option( options[:new] ).merge( actions_from_option(options[:additional_new]) )
```
Comment (inkdeep) [inline lib/declarative_authorization/in_controller.rb]: I'm removing this change. I  unwrapped the line when I was working my way through the code and it snuck in.

File: lib/declarative_authorization/in_controller.rb (hunk 1 above)
Comment (inkdeep) [inline lib/declarative_authorization/in_controller.rb:373]: Will setting  :member => {:photo => :read}  will make :photo the _**only**_ :member action in the controller? Meaning :edit, :update, :destroy become collection actions due to not being in the :member hash?  I don' think this is correct but I want to clarify. 
filter_resource_access method merges the options hash with the options argument hash?

Ah! ignore me. The default value is an array which are overwritten with a hash if set. Doh!

**Pull request closed without merging**